cms/startup.py

```diff
@@ -21,6 +23,11 @@ def run():
 
     add_mimetypes()
 
+    # Monkey patch the keyword function map
```

Comment:
Can you explain what's going on here?

_Why_ are we monkey patching the function map?